packages/data-stores/src/automated-transfer-eligibility/selectors.ts

```diff
@@ -18,77 +16,73 @@ export const getAutomatedTransferEligibility = (
 	return state[ siteId ];
 };
 
-export const getEligibilityHolds = createRegistrySelector(
-	( select: TransferSelectFn ) =>
-		( state: State, siteId: number | null ): TransferEligibilityError[] | null => {
-			const transferEligibility = select( STORE_KEY ).getAutomatedTransferEligibility( siteId );
-
-			if ( ! transferEligibility ) {
-				return null;
-			}
-
-			const { errors } = transferEligibility;
-
-			if ( ! errors ) {
-				return null;
-			}
-
-			return errors.map( ( { code } ) => {
-				return statusMapping[ code ] ?? '';
-			} );
-		}
-);
-
-export const getEligibilityWarnings = createRegistrySelector(
-	( select: TransferSelectFn ) =>
-		( state: State, siteId: number | null ): TransferEligibilityWarning[] | null => {
-			const transferEligibility = select( STORE_KEY ).getAutomatedTransferEligibility( siteId );
-
-			if ( ! transferEligibility ) {
-				return null;
-			}
-
-			const { warnings } = transferEligibility;
-
-			if ( ! warnings ) {
-				return null;
-			}
-
-			// combine plugin and theme warnings into one list
-			const combined = Object.values( warnings ).flat();
-
-			return combined.map( ( { description, name, supportUrl, id } ) => ( {
-				id,
-				name,
-				description,
-				supportUrl,
-			} ) );
-		}
-);
-
-export const getNonSubdomainWarnings = createRegistrySelector(
-	( select: TransferSelectFn ) =>
-		( state: State, siteId: number | null ): TransferEligibilityWarning[] | null => {
-			const eligibilityWarnings = select( STORE_KEY ).getEligibilityWarnings( siteId );
-
-			if ( ! eligibilityWarnings ) {
-				return null;
-			}
-
-			// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-			// @ts-ignore Until createRegistrySelector is typed correctly
-			return eligibilityWarnings?.filter( ( { id } ) => id !== 'wordpress_subdomain' ) ?? null;
-		}
-);
-
-export const getWpcomSubdomainWarning = createRegistrySelector(
-	( select: TransferSelectFn ) =>
-		( state: State, siteId: number | null ): TransferEligibilityWarning | null => {
-			const eligibilityWarnings = select( STORE_KEY ).getEligibilityWarnings( siteId );
-			// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-			// @ts-ignore Until createRegistrySelector is typed correctly
-			const warning = eligibilityWarnings?.find( ( { id } ) => id === 'wordpress_subdomain' );
-
-			return warning || null;
-		}
-);
+export const getEligibilityHolds = (
+	state: State,
+	siteId: number | null
+): TransferEligibilityError[] | null => {
+	const transferEligibility = getAutomatedTransferEligibility( state, siteId );
```

Comment:
This is an internal selector in the same file, so the registry selector pattern isn't needed here. (Same for the others!)